packages/ui-components/scripts/generate-illustrations.js

```diff
@@ -59,15 +60,45 @@ async function main() {
 		const className = upperFirst(camelCase(key));
 		const parsedData = parse(defaultItem.data);
 
+		let htmlContent = toHTML(getRoot(parsedData), { allowDangerousHtml: true });
+	
+		const styleMatches = htmlContent.match(/style="[^"]+"/g) || [];
```

Comment:
This is due to the "mask-type" style, right? 😞

Reply:
Yes, and now we can handle more styles also if needed.